components/Ripple/src/MaterialState.h

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
```

Comment:
This file looks like something that should live in its own directory as a standalone component.